### ISSUE TITLE
bash used but not installed?

### ISSUE THREAD

**Issue by inphobia** [netdisco/netdisco-docker#20]
https://github.com/netdisco/netdisco-docker/blob/42ff452a1ee2398a0705607ca3cced138aa3cd0c/netdisco-base/wait_to_start.sh#L1

came about this file while fooling around with our containers. now, this file is found in our container images like netdisco-backend & -web, but those systems are lean and include ash, not bash, so this script will never run on the container images.

now, i have no idea where this file is used, i'm guessing somewhere in the container image build process, so this might not even be relevant.

bottom line: this seems to be a simple script, perhaps bash can be change to `#!/bin/sh` instead?

**Comment by LBegnaud** [comment]
doesn't look like it's used anywhere, but it is a slick little script that I would hate to see go. Unless we have a need to save this 1KB, can we decide just to leave it?

**Comment by inphobia** [comment]
yeah, but if it doesn't need bash specific things can't we just use `#!/bin/sh` ?

**Comment by LBegnaud** [comment]
But if it's not even in use... does it matter?

**Comment by inphobia** [comment]
it's mentioned here: https://github.com/netdisco/netdisco-docker/blob/9538ffbf18869e05b21af54e41322d903a90c976/netdisco-base/Dockerfile#L76

but i guess that's used during some kind of container build process?

and does it matter, i guess not really, but being a zsh guy it's a bummer to think bash it the only shell 🙂 . the rest of the dockerfile above uses ash for as shell btw.

**Comment by LBegnaud** [comment]
yea that's just copying the file into the container
